### ISSUE TITLE
Want to add content in STL section

### ISSUE THREAD

**Issue by pra9patil** [UjjwalSharma01/checklist#53]
Hey @UjjwalSharma01 sir
In STL/Containers(vector) I want to add contet with code snipest and functions used to retrive ,delete , modify and capcity 
only for vector container in because it will take lots of time to write down every function for every container 
will you please assign me this issue so I can work on it

**Comment by UjjwalSharma01** [comment]
Go ahead

**Comment by pra9patil** [comment]
Sir I go through the hactoberfest website,
in oreder to accepted by hactoberfest 
hactoberfest-accepeted label is mandatory so will you please add it .

**Comment by UjjwalSharma01** [comment]
I'm well aware of all the guidelines, hacktoberfest-accepted label will be added after you PR is merged

**Comment by pra9patil** [comment]
Sir @UjjwalSharma01  , I have one query , where I have to create new checklist under container checklist with name of vector or just I add the content about vector with their name like you did in c++ basics please guide me here

**Comment by pra9patil** [comment]
![Screenshot (91)](https://github.com/UjjwalSharma01/checklist/assets/130309887/3b4686a9-fcc6-4a9e-9af2-0acbb666c676)
![Screenshot (90)](https://github.com/UjjwalSharma01/checklist/assets/130309887/a7c0a2fa-13ac-4bcc-a4ef-356e8e8407ff)

after adding this I am facing issue which is in the scond screen shot will please guide me on that how I supposed to fix this issue so I can work further

**Comment by pra9patil** [comment]
Check box and other content in that checkbox section is mixing with content which I added in vector dropdown

**Comment by UjjwalSharma01** [comment]
> Check box and other content in that checkbox section is mixing with content which I added in vector dropdown 

We will fix this in further updates

For the time being don't include this as the subtopic of an already existing checklist

Add this as a complete topic

**Comment by pra9patil** [comment]
Then I have to add only topic name and small content because if given height get extended then it will not work give same issue

**Comment by UjjwalSharma01** [comment]
nope, i meant to say, instead of adding content under container tag, create a separate list to avoid this, we are working on the issue
